inst/examples/404.html

```diff
@@ -0,0 +1,342 @@
+<!DOCTYPE html>
```

Comment:
I may have missed something but why do we need a 404.html at the root of the book project ? 

The 404.html should be created in the output folder at build time using the default content. 

There should not be one in the source, unless we want to customize and it would be `_404.Rmd` or `_404.md`

Reply:
It auto-created that for me- I'll remove but won't it just regenerate when I rebuild the book?

Reply:
It should be created then moved in the `_book/` directory. So `inst/examples/_book` locally which should be ignored for git. 

I tried it and it works this way for me. If not for you, there could be a bug... 

And it is already live this way since we merged the PR 

https://bookdown.org/yihui/bookdown/404.html

Reply:
welp

Reply:
I'll remove from this PR, but all I did was serve the book 👼

Reply:
BTW why doesn't this work then? https://bookdown.org/yihui/bookdown/an-unwritten-chapter.html

Reply:
It doesn't work because it is hosted on rstudio connect (bookdown.org)

I'll look tomorrow add this while serving the book. I did not try serving and I may have missed something !

Reply:
oh right! Connect. I removed the file and re-rendered, but it was not re-created 🤷‍♀️